kubernetes-informer/src/main/java/io/micronaut/kubernetes/client/informer/package-info.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2017-2022 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * Contains classes specific to Kubernetes informer.
+ *
+ * @author Nemanja Mikic
+ * @since 3.4.1
+ */
+
+@Configuration
+@Requires(env = Environment.KUBERNETES)
```

Comment:
this is a breaking change is not it? Is this necessary?

Reply:
This change is from `master` branch